src/commands/apps/lib/format-app.ts

```diff
@@ -15,7 +15,7 @@ export function reportApp(app: models.AppResponse): void {
       ["Owner Display Name", "owner.displayName"],
       ["Owner Email", "owner.email"],
       ["Owner Name", "owner.name"],
-      ["Azure Subscription ID", "azureSubscriptionId"],
+      ["Azure Subscription ID", "azureSubscription.subscriptionId"],
```

Comment:
Is it a bug fix? I would like to know if this is relevant to the `orgs apps create` command .. (don't think so)

Reply:
When the `orgs apps create` command is done creating the app, it writes the app information out to the console using this method, the same way `apps create` does. That method has a bug in it, which I'd like to fix as part of adding the new command.